content/academy/web_scraping_for_beginners/challenge/scraping_amazon.md

```diff
@@ -0,0 +1,226 @@
+---
+title: Scraping Amazon
+description: Before you build your first web scraper with Crawlee, it is important to understand the concept of modularity in programming.
+menuWeight: 4
+paths:
+- web-scraping-for-beginners/challenge/scraping-amazon
+---
+
+# Scraping Amazon
+
+In our quick chat about modularity, we finished the code for the results page, and added a request for each product to the crawler's **RequestQueue**. Here, we just need to scrape the description, so it shouldn't be too hard:
+
+```JavaScript
+// routes.js
+
+// ...
+
+router.addHandler(labels.PRODUCT, async ({ $, crawler, request }) => {
+    const { data } = request.userData;
+
+    const element = $('div#productDescription');
+
+    const description = element.text().trim();
+
+    console.log(description); // works!
+});
+```
+
+> If you are sometimes getting an error along the lines of **RequestError: Proxy responded with 407**, don't worry, this is totally normal. The request will retry and succeed.
+
+Great! But wait, where do we go from here? We need to go to the offers page next and scrape each offer, but how can we do that? Let's take a small break from writing the scraper and open up [Proxyman]({{@link tools/proxyman.md}}) to analyze requests which we can't see inside the network tab, then we'll click the button on the product page that loads up all of the product offers:
```

Comment:
What's the purpose of using Proxyman here? All requests should be visible in the Network tab.

Reply:
need to add context to this